modules/social_features/social_group/modules/social_group_members_export/src/Plugin/Action/ExportMember.php

```diff
@@ -54,7 +54,7 @@ public function access($object, AccountInterface $account = NULL, $return_as_obj
    * @see social_user_export_file_download()
    */
   protected function generateFilePath() : string {
-    $hash = md5(microtime(TRUE));
+    $hash = md5((string) microtime(TRUE));
```

Comment:
Same here, microtime(FALSE) should return a string

Reply:
Added changes.